memecity.game/Assets.h

```diff
@@ -22,6 +22,13 @@ namespace assets {
 	namespace spritesheets {
 		static constexpr Asset HUMAN_MALE_1 = "Spritesheets/SpriteSheet.png";
 	}
+	namespace advertisements {
+		static constexpr Asset ADVERTISEMENT_ONE = "Advertisements/BurgerKing.png";
+		static constexpr Asset ADVERTISEMENT_TWO = "Advertisements/BananaWho.png";
+		static constexpr Asset ADVERTISEMENT_THREE = "Advertisements/CocaCola.png";
+
+		const static std::vector<std::string> ads{ ADVERTISEMENT_ONE, ADVERTISEMENT_TWO, ADVERTISEMENT_THREE };
```

Comment:
Ik denk dat of een methode `get_ad`, en/of het hernoemen naar `ADS` netter zou zijn,